src/azure-cli/HISTORY.rst

```diff
@@ -13,7 +13,11 @@ Release History
 
 **RBAC**
 
-* ad signed-in-user: fix a crash on listing owned objects 
+* ad signed-in-user: fix a crash on listing owned objects
+
+**HDInsight**
```

Comment:
[nit] Please alphabetize this service entry in the changelog.

Reply:
> [nit] Please alphabetize this service entry in the changelog.

Have updated